src/usage/UsageToday.tsx

```diff
@@ -0,0 +1,119 @@
+// Libraries
+import React, {FC} from 'react'
+import {useDispatch, useSelector, connect} from 'react-redux'
+import {
+  ComponentSize,
+  FlexBox,
+  AlignItems,
+  FlexDirection,
+  Panel,
+} from '@influxdata/clockface'
+
+// Components
+import UsageDropdown from 'src/usage/UsageDropdown'
+import BillingStatsPanel from 'src/usage/BillingStatsPanel'
+import TimeRangeDropdown from 'src/shared/components/TimeRangeDropdown'
+import GraphTypeSwitcher from 'src/usage/GraphTypeSwitcher'
+
+// Constants
+import {GRAPH_INFO} from 'src/usage/Constants'
+import {getTimeRange} from 'src/dashboards/selectors'
+import {setTimeRange} from 'src/timeMachine/actions'
+import {useUsage} from 'src/usage/UsagePage'
+
+// Types
+import {TimeRange} from 'src/types'
+
+interface StateProps {
+  selectedUsageID: string
+}
+
+type Props = StateProps
+
+const UsageToday: FC<Props> = ({selectedUsageID}) => {
```

Comment:
I think you can just do `FC<StateProps>` and get rid of `type Props = StateProps`